src/parsing/test/testing.ts

```diff
@@ -152,20 +155,23 @@ export function toEqualDefinition(actualResponse, testDefinition: TestCase) {
     }
 
     if (actualResponse.suggestKeywords) {
-        const weightFreeKeywords = [];
+        const weightFreeKeywords: ParseResult['suggestKeywords'] = [];
         actualResponse.suggestKeywords.forEach(keyword => {
-            weightFreeKeywords.push(keyword.value);
+            if (typeof keyword !== 'string') {
+                // @ts-ignore
```

Comment:
We always add comments when we ignore something. Please write that this fill will be obsolete in 1-2 weeks, because we're rewriting tests. It's important, because without extra comments code starts to look like 💩, but if you see a comment, you start to understand why it's necessary, and your code looks like 🏅 

It should be fixed at each `@ts-ignore`.